playbooks/openstack/advanced-configuration.md

```diff
@@ -327,17 +327,34 @@ The `openshift_openstack_required_packages` variable also provides a list of the
 prerequisite packages to be installed before to deploy an OpenShift cluster.
 Those are ignored though, if the `manage_packages: False`.
 
-## Multi-master configuration
+## API and Router Load Balancing
+
+If you run multiple master or infra nodes, you want to put a load balancer in
+front of them.  We can configure an Octavia or Neutron LBaaS V2 load balancer
+automaticaly.
+
+All you need to do is put this in your `inventory/group_vars/all.yml`:
+
+    openshift_openstack_use_lbaas_load_balancer: true
+
+This will create a single load balancer with listeners for the API and router
+nodes.
+
+
+If you don't want to use OpenStack's LBaaS, we can create and configure a VM
+running HAProxy and use that:
+
+You can get its IP address by setting:
+
+    openshift_openstack_use_vm_load_balancer: true
+
+In either case, you can get the IP addresses for the API and routers by
```

Comment:
nit: is it in fact the IP of LB (frontend) for those? Let's please clarify,

Reply:
Yeah, it's the IP of the public LB. I've updated the docs to make this clearer.